src/views/Main.js

```diff
@@ -4,7 +4,22 @@ import Layout from '../components/Layout'
 
 export default () => (
   <Layout>
-    <h1>Main View</h1>
-    <Link to='/second'>Second view</Link>
+    <section>
+      <h1>Heading</h1>
```

Comment:
Do wen want these in the master? I know you've used them for testing, but are they necessary for our main code-base? 😄